roles/ceph-rgw/tasks/docker/main.yml

```diff
@@ -4,3 +4,6 @@
 
 - name: include start_docker_rgw.yml
   include: start_docker_rgw.yml
+
+- name: include rgw_pool_pgs_docker.yml
+  include: rgw_pool_pgs_docker.yml
```

Comment:
add:
```
when:
  - create_pools is defined
  - containerized_deployment
```

Reply:
This task is inside tasks/docker/.   containerized_deployment has already been check here: https://github.com/jtudelag/ceph-ansible/blob/a551faca4036a0418d3c4cd7216281c856610203/roles/ceph-rgw/tasks/main.yml#L48

Reply:
true